tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es2020",
     "module": "commonjs",
-    "lib": ["es2020"],
+    "lib": ["es2020", "dom"],
```

Comment:
This was added for `fetch-mock-jest`. Apparently it's known because that library also supports mocking `cross-fetch`.

Reply:
I would avoid adding `dom` to lib since this is a Node.js library and I wouldn't want `dom` methods to be added to the build.

Would it make sense to create a separate `tsconfig` for tests?

Reply:
I can take a look at that. Not sure how difficult it would be.

Reply:
Note: it's the `lint:tsc` command that fails that requires it. I suppose we'd have to separate the tests into a separate command to make that happen.

Reply:
Added separate tsconfig.test.json and updated the lint:tsc command to utilize it.

Reply:
Thanks